pcsx2/R3000A.cpp

```diff
@@ -222,7 +222,7 @@ __ri void iopEventTest()
 	{
 		// start the next branch at the next counter event by default
 		// the interrupt code below will assign nearer branches if needed.
-		if (psxNextCounter < (psxRegs.iopNextEventCycle - psxNextsCounter))
+		if (static_cast<u32>(psxNextCounter) < (psxRegs.iopNextEventCycle - psxNextsCounter))
```

Comment:
This one's a bit suss, I think the right side should be casted to signed instead (it's saying is the delta less than the time to the next event).

Current behaviour w/ the warning is to do an unsigned comparison, but I'm not sure if that's correct.

Reply:
yeah, in the event it's overrun, you don't want it waiting 2 billion cycles :)

Reply:
All good now?

Reply:
Yes that's better

Reply:
Was more of just in case it's not big enough, but sure can switch to s32.

Reply:
Well the others are only u32 so it can never be bigger than 32bit